### PR TITLE
Fix #148: Adding .editorconfig and other VSCode recommendations

### LINKED ISSUE #148: Maintaining line endings across OS and Editors sucks the joy out of development
We've got a couple of PRs out here to address this but wanted to throw up a quick issue to briefly describe what we're trying to achieve and provide a link between the two PRs.

Depending on your operating system and editor Git or your editor might choose to end all of your lines with either a carriage return and a line feed (CRLF) or just a line feed (LF). Which no one really cares about and it's hard to spot the difference until you start a pull request. If the line endings on a file have been changed from CRLF to LF or vice versa the resulting git diff will show that you have modified every line in the file instead of just highlighting the period you maybe added to the end of a Write-Host statement.

We are wanting to add a few things to our shared development environment to hopefully make us all forget that this was ever even a struggle.

Thanks @jeremymcgee73 for getting this started! 😎😎😎

### PR DESCRIPTION
**Pull request by Tiberriver256**
- [.editorconfig](https://editorconfig.org/) to ease the headache of odd formatting issues across editors and OSes
- formatOnSave to true in settings.json to ensure PowerShell code is formatted consistently and without having to think about it
- Added the following recommended extensions for VSCode
   - [ms-vscode.PowerShell](https://marketplace.visualstudio.com/items?itemName=ms-vscode.PowerShell) of course!
   - [EditorConfig.EditorConfig](https://github.com/editorconfig/editorconfig-vscode) to ensure VSCode is configured to behave according to the .editorconfig file
- Fixed a little bug in the tasks.json file. We were missing a slash which turned a comment into a not comment...

Related issue: #148

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = crlf
+charset = utf-8
+trim_trailing_whitespace = false
+insert_final_newline = false
```
Comment (tylerl0706) [inline .editorconfig]: nitpick: new line at EOF is usually best practice with git

Comment (tylerl0706) [inline .editorconfig]: Testing the new suggestion feature

```suggestion
insert_final_newline = false

```

Comment (Tiberriver256) [reply .editorconfig]: Handy. I have a big apply suggestion button. Although I think you meant to change it to true.

Comment (tylerl0706) [reply .editorconfig]: Ah! Yes!

**Pull request merged**